cob_bringup/robots/cob4-16.xml

```diff
@@ -131,20 +131,20 @@
 			<arg if="$(arg sim)" name="cartesian_control" value="true"/>
 		</include>
 
-		<!--include file="$(find cob_bringup)/components/canopen_generic.launch">
+		<include file="$(find cob_bringup)/components/canopen_generic.launch">
 			<arg name="robot" value="$(arg robot)"/>
 			<arg name="component_name" value="arm_left"/>
-			<arg name="can_device" value="can3"/>
+			<arg name="can_device" value="can2"/>
```

Comment:
why are can devices switched for the arms? in cob4-5 and cob4-8 we have 
- arm_right: can2
- arm_left: can3

Reply:
this is how the robot hw is set up...